bridgesync/downloader.go

```diff
@@ -44,6 +44,8 @@ var (
 	claimMessageEtrogMethodID    = common.Hex2Bytes("f5efcd79")
 	claimAssetPreEtrogMethodID   = common.Hex2Bytes("2cffd02e")
 	claimMessagePreEtrogMethodID = common.Hex2Bytes("2d2c9d94")
+
+	BridgeAssetMethodID = common.Hex2Bytes("cd586579")
```

Comment:
This const can be private, WDYT?
```suggestion
	bridgeAssetMethodID = common.Hex2Bytes("cd586579")
```

Reply:
It's exported because is it required in test code,

Reply:
The test code belongs to the same package, `bridgesync`, as the code being tested, so it doesn't seem necessary to export it. Wdyt?

Reply:
Sadly the package is `bridgesync_test` ([here](https://github.com/agglayer/aggkit/blob/e3fe3c755ebfac596a8410183985b7ed603fd270/bridgesync/e2e_test.go#L1)) so everything is imported